msal/src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -901,6 +901,11 @@ internal async Task RemoveAsync(string authority, bool validateAuthority, IAccou
 
         internal void RemoveMsalAccount(IAccount account, ISet<string> environmentAliases, RequestContext requestContext)
         {
+            if (account.HomeAccountId == null)

```

Comment:
>if (account.HomeAccountId == null) [](start = 12, length = 34)

is this properly covered by utests? 
Also should probably at least be logged as info?

Reply:
Adal v3 to Msal v2 migration is not covered by unit test, this integration project is only one which has references to adal an msal so this scenario can be tested